package.json

```diff
@@ -78,6 +78,7 @@
     "less-vars-to-js": "^1.3.0",
     "localforage": "^1.7.4",
     "lodash": "^4.17.13",
+    "lru-memoize": "^1.1.0",
```

Comment:
Using `lru-memoize` instead of `lodash`'s implementation because this one allows us to cache more than one set of parameters and also it caches by all parameters received (as opposed to lodash which caches by the first parameter only)